src/openrct2/actions/LoadOrQuitAction.hpp

```diff
@@ -0,0 +1,74 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2018 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include "../Context.h"
+#include "../OpenRCT2.h"
+#include "GameAction.h"
+
+enum class LoadOrQuitModes : uint8_t
+{
+    OpenSavePrompt,
+    CloseSavePrompt
+};
+
+DEFINE_GAME_ACTION(LoadOrQuitAction, GAME_COMMAND_LOAD_OR_QUIT, GameActionResult)
+{
+private:
+    uint8_t _mode{ 0 };
+    uint8_t _savePromptMode{ 0 };
```

Comment:
I think this should be a bool.

Reply:
Turns out its of type enum `PROMPT_MODE` so I've kept it as `uint8_t` and add the appropriate constants.

Reply:
I see, since it was either 0 or 1 I sort of assumed its nothing more than that.